TicTacToe.Foundation/Figure/FigureType.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace TicTacToe.Foundation.Figure
+{
+    enum FigureType
```

Comment:
1. What is the default access modifier?
2. Always specify access modifiers explicitly.